Boa.Constrictor/Utilities/MessagesWrapper.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+
+namespace Boa.Constrictor.Utilities
+{
+    /// <summary>
+    /// Provides methods for wrapping long messages string.
+    /// </summary>
+    public static class MessagesWrapper
+    {
+        /// <summary>
+        /// Wraps long lines of messages with provided limit of text to display as wrapped line.
+        /// </summary>
+        /// <param name="message">The Message on which word wrap will apply.</param>
+        /// <param name="limit">The limit of characters to be applied.</param>
+        /// <returns></returns>
+        public static List<string> GetWrappedMessage(string text, int limit)
```

Comment:
You must add a `using` statement to use `List<string>`.

Here's the error message: 
```
D:\a\boa-constrictor\boa-constrictor\Boa.Constrictor\Utilities\MessagesWrapper.cs(16,23): error CS0246: The type or namespace name 'List<>' could not be found (are you missing a using directive or an assembly reference?) [D:\a\boa-constrictor\boa-constrictor\Boa.Constrictor\Boa.Constrictor.csproj]
```

Reply:
@AutomationPanda : let me fix this.

Reply:
@AutomationPanda : Sorry for the delay, this should be fixed now.